src/Integration.Vsix.UnitTests/CFamily/MsvcDriverTest.cs

```diff
@@ -436,7 +437,34 @@ public void Version()
             req.Predefines.Should().NotContain(X86_MACROS);
             req.Predefines.Should().Contain(X64_MACROS);
             req.Predefines.Should().Contain("#define _HAS_CHAR16_T_LANGUAGE_SUPPORT 1\n");
+            req.Predefines.Should().NotContain("#define _HAS_CONDITIONAL_EXPLICIT 0\n");
 
+            req = MsvcDriver.ToRequest(new CFamilyHelper.Capture[] {
+                new CFamilyHelper.Capture()
+                {
+                    Executable = "",
+                    StdOut = "",
+                    CompilerVersion = "19.25.28610",
```

Comment:
There are quite a few test cases that aren't covered
e.g. major < 19, minor > 23,
major = 19. minor = 23
major = 19, minor > 23
major > 20, minor = < 23

This would be a good place to use a data-driven test.

Reply:
As discussed offline, I'll push a commit with a data-driven test.